addons/mail/static/src/components/message_list/message_list.js

```diff
@@ -395,8 +395,8 @@ class MessageList extends Component {
             return;
         }
         if (
-            this.threadView.lastVisibleMessage &&
-            (message.id < this.threadView.lastVisibleMessage.id)
+            this.threadView.lastVisibleMessageOfThreadCache &&
+            (message.id < this.threadView.lastVisibleMessageOfThreadCache.id)
```

Comment:
This was a typo?

Reply:
Nvm js field rename in stable.
Could have kept the previous name and give a proper name in master.

But I guess since we are only 2 weeks in this is acceptable... Wouldn't do this in 3 months for sure though.